direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaAccessor.java

```diff
@@ -228,19 +235,20 @@ private void configure(Map<String, Object> cfg) {
 
   Properties createProps() {
     Properties props = new Properties();
+    props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, this.getUri().getAuthority());
     for (Map.Entry<String, Object> e : cfg.entrySet()) {
       if (e.getKey().startsWith(KAFKA_CONFIG_PREFIX)) {
         props.put(e.getKey().substring(KAFKA_CONFIG_PREFIX.length()), e.getValue().toString());
       }
     }
+    props.putIfAbsent(ProducerConfig.ACKS_CONFIG, DEFAULT_PRODUCER_ACK_SETTINGS);
+    props.putIfAbsent(ProducerConfig.BATCH_SIZE_CONFIG, DEFAULT_PRODUCER_BATCH_SIZE_SETTINGS);
```

Comment:
Why do we override Kafka defaults for these settings? `linger.ms` seems to be related to batch.size, so if we have a reason to override batch.size, should we override `linger.ms` as well?

Reply:
No idea, I just make those options configurable because it was hardcoded in origin code, it just follows original settings. What do you propose?

Reply:
Ah, I understand. That seems to be a bug. Maybe it is enough to remove the hard-coded "defaults"? Both values should have the exact same defaults.

Reply:
I didn't know it was default value, in that case we can simply remove it.

Reply:
removed.